packages/core/src/vuePlugins/actions/addInterceptor.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  CoreGetters,
+  CoreState,
+  PolarActionContext,
+} from '@polar/lib-custom-types'
+
+export function addInterceptor({
+  getters,
+}: PolarActionContext<CoreState, CoreGetters>) {
+  const { interceptorUrlRegex } = getters.configuration
+  const { fetch: originalFetch } = window
+
+  // If interceptors for XMLHttpRequest or axios are needed, add them here
+  window.fetch = (resource, originalConfig) => {
+    let config = originalConfig
+
+    if (
+      interceptorUrlRegex &&
+      typeof resource === 'string' &&
+      resource.match(interceptorUrlRegex)
+    ) {
+      config = {
+        ...originalConfig,
+        headers: {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          Authorization: `Bearer ${getters.oidcToken}` || '',
```

Comment:
```suggestion
    if (
      getters.oidcToken &&
      interceptorUrlRegex &&
      typeof resource === 'string' &&
      resource.match(interceptorUrlRegex)
    ) {
      config = {
        ...originalConfig,
        headers: {
          // eslint-disable-next-line @typescript-eslint/naming-convention
          Authorization: `Bearer ${getters.oidcToken}`,
```
Didn't work, I suppose you meant that.

Reply:
`Bearer ${getters.oidcToken}` || ''`

This statement never falls back to '', no matter what value `getters.oidcToken` holds.

Reply:
Adjusted 7abee88c097e0c2b484e046315bd2acffe12e402